sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -72,6 +72,10 @@
 def default_encoder(obj):
   if isinstance(obj, decimal.Decimal):
     return str(obj)
+  elif isinstance(obj, bytes):
```

Comment:
This will always evaluate to true for strings on Python 2. Add an `if` as in previous version to avoid an extra operation?

Reply:
This function is only called on objects that fail to be serialized with the standard `json.dumps`. Strings do not make it into this function.

Reply:
See https://docs.python.org/3/library/json.html#json.dump

Reply:
Thanks for explanation.